functions/sort_hash.pp

```diff
@@ -0,0 +1,4 @@
+function jira::sort_hash(Hash $input) {
```

Comment:
I think you can actually specify the output type and document it as a private API
```suggestion
# @summary Return a sorted hash
# @api private
function jira::sort_hash(Hash $input) >> Hash {
```